helical/models/geneformer/geneformer_utils.py

```diff
@@ -132,9 +170,10 @@ def load_model(model_type, model_directory):
         model = BertForMaskedLM.from_pretrained(
             model_directory, output_hidden_states=True, output_attentions=False
         )
-    # put the model in eval mode for fwd pass
+    # put the model in eval mode for fwd pass and load onto the GPU if available
     model.eval()
-    # model = model.to("cuda:0")
+    if torch.cuda.is_available():
+        model = model.to("cuda:0")
```

Comment:
I think we should use the device here that is set in the config

Reply:
I agree but this function is only used as part of testing, so it's not linked to a user-set config. It's a utils function that we don't use anywhere else. But we could maybe pass a device arg instead so it is reusable anywhere else?